src/harness/harness.ts

```diff
@@ -1897,10 +1910,16 @@ namespace Harness {
             let currentFileOptions: any = {};
             let currentFileName: any;
             let refs: string[] = [];
+            let symlinks: [string, string][] | undefined;
 
             for (const line of lines) {
                 const testMetaData = optionRegex.exec(line);
-                if (testMetaData) {
+                const linkMetaData = linkRegex.exec(line);

```

Comment:
Why match this as both and option and a link? I would recommend moving the `const testMetaData …` line into the else branch.